data/sql/updates/pending_db_world/rev_1677249657204460100.sql

```diff
@@ -0,0 +1,9 @@
+-- Thule Ravenclaw - remove ranged movement
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 1947;
+
```

Comment:
Doesn't need these 2 lines.
It's already SmartAI in the template 😛

Reply:
Going by sql standards (the example there does the same with already pre-existing SmartAI).
But I don't know how trivial or not is to do that.